src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/AmqpExtIntegrationTest.php

```diff
@@ -76,20 +76,14 @@ public function testItSendsAndReceivesMessages()
         $this->assertEmpty(iterator_to_array($receiver->get()));
     }
 
-    /**
-     * @group legacy
-     * ^ for now, deprecation errors are thrown during serialization.
-     */
```

Comment:
This test was broken because it uses an incomplete symfony serializer (see https://github.com/symfony/symfony/runs/3097443193)